### ISSUE TITLE
LinkedIn Outreach

### ISSUE THREAD

**Issue by bbertucc** [EqualifyEverything/equalify-outreach#2]
Jesse Sookne initiated a LinkedIn outreach project on the a11y slack. Here is a link to that Slack thread: https://web-a11y.slack.com/archives/C0241L0TMGD/p1717526814051989

Hopefully, Team Equalify can join this LinkedIn campaign.

Here is what Jesse is thinking:
- I start a DM thread for the ~6 of us interested. (Alternatively, we coordinate everything here in [#freelance](https://web-a11y.slack.com/archives/C0241L0TMGD) so others can see and join later if they want to).
- We share the frequency / cadence we intend to post, time of day that’s convenient for us to post, and themes we’re likely to cover.
- We pair up based on best match for the three criteria above.
- We commit to working with our partner to promote each post within the first few minutes it goes up. Maybe we commit for a specific timeframe initially, e.g. for a month, so we can see how the process goes.
- We also jump in to promote posts from the other pairs as/when we can. No specific commitment here

I'll continue to update this ticket with updates from Jesse.

**Comment by bbertucc** [comment]
@kevinandrews1 - Did you want to be part of this too?

**Comment by kevinandrews1** [comment]
Jesse, @bbertucc, and I met this afternoon. Very good discussion. Here's what happened:
- Engage immediately after a post is published to emphasize its importance.
- Encourage comments on new posts; LinkedIn's algorithm favors comments.
- Monitor dwell time (how long the post remains on screen) to gauge reading interest.
- Track profile views generated by posts; LinkedIn prefers posts that drive profile visits.
- Establish a consistent posting time.
- Use Buffer, if you want, to help automate post scheduling at specific times.
- Share insights and data about LinkedIn’s engagement strategies with the team as we proceed in the ocming weeks.
- Engagement responsibilities:
    - Kevin: Engage Monday and Wednesday, at least twice a week, at noon Eastern/11 Central.
    - Blake: Same.
    - Jesse: Engage five days a week.
- Blake will seek feedback on the a11y-marketing channel without requiring responses.
